src/components/resizable_container/resizable_panel.test.tsx

```diff
@@ -57,5 +57,17 @@ describe('EuiResizablePanel', () => {
 
       expect(container).toMatchSnapshot();
     });
+
+    test('tabindex', () => {
```

Comment:
super minor casing nit
```suggestion
    test('tabIndex', () => {
```